SonarQube.Client/Models/ServerSentEvents/ClientContract/IIssueChangedServerEvent.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarQube.Client.Models.ServerSentEvents.ServerContract;
+
+namespace SonarQube.Client.Models.ServerSentEvents.ClientContract
+{
+    /// <summary>
+    /// Represents an event that was created from <see cref="IIssueChangedSqServerEvent"/>
+    /// </summary>
+    public interface IIssueChangedServerEvent : IServerEvent
```

Comment:
We might not need this layer of abstraction, at least not for now.
My rational was that ISqServerEvent is the raw data as it is sent from SQ, which can change between versions. This is the object that SLVS subscribes to.